packages/plugin-compiler/src/test/kotlin/io/realm/kotlin/compiler/GenerationExtensionTest.kt

```diff
@@ -89,7 +89,7 @@ class GenerationExtensionTest {
         fun assertGeneratedIR() {
             val outputFile = File("${outputDir()}/main/01_AFTER.ValidateIrBeforeLowering.ir")
             stripInputPath(outputFile, fileMap)
-            val expected = File("${expectedDir()}/00_ValidateIrBeforeLowering.ir").readText()
+            val expected = File("${expectedDir()}/01_AFTER.ValidateIrBeforeLowering.ir").readText()
```

Comment:
What has changed here?

Reply:
The output file generated from the compiler was changed a while ago. This just renames the `expected` files to the same name that the compiler generates